src/components/AlertModal/AlertModal.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { FunctionComponent, useMemo } from "react";
+import { Modal, StyleSheet, Text, View } from "react-native";
+import AlertLogo from "../../../assets/icons/alert.svg";
+import { DarkButton } from "../Layout/Buttons/DarkButton";
+import { SecondaryButton } from "../Layout/Buttons/SecondaryButton";
+import { DangerButton } from "../Layout/Buttons/DangerButton";
+import { size } from "../../common/styles";
+
+const styles = StyleSheet.create({
+  centeredView: {
+    flex: 1,
+    width: "100%",
+    justifyContent: "center",
+    alignItems: "center",
+    alignSelf: "center",
+    backgroundColor: "rgba(0,0,0,0.6)"
+  },
+  modalView: {
+    marginBottom: 0,
+    paddingTop: 24,
+    width: 280,
+    backgroundColor: "white",
+    borderRadius: size(1),
+    shadowColor: "#000",
+    shadowOffset: {
+      width: 0,
+      height: 2
+    },
+    shadowOpacity: 0.25,
+    shadowRadius: 3.84,
+    elevation: 5
+  },
+  alertIcon: {
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "center",
```

Comment:
Don't need these styles since they don't affect anything
```suggestion
```

Reply:
You are right. Removed